src/main/resources/templates/dev-hub/authentication.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+
+<html lang="en-GB" layout:decorate="~{layouts/baseLayout}"
+    xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+    xmlns:th="http://www.thymeleaf.org">
+
+<head>
+<title th:utext="${templateTitle}"></title>
+</head>
+
+<body>
+
+    <div layout:fragment="content">
+        <h1 class="govuk-heading-xl">Authentication</h1>
+        <h2 class="govuk-heading-l">API key authentication</h2>
+
+        <p class="govuk-body">The Companies House API requires
+            authentication credentials, in the form of an API key, to be
+            sent with each request.</p>
+
+        <p class="govuk-body">
+            To obtain an API key, go to <a href=https://www.example.com />Your
+            applications</a> and register the application with the Companies
+            House Developer Hub as an
+            <code>API Key</code>
+            application. This will allocate a unique key to the
+            application which can be sent with any
+            <code>GET</code>
+            request for a public resource served by the Companies House
+            API.
+        </p>
+
+        <h3 class="govuk-heading-m">Sending the API key</h3>
+        <p class="govuk-body">
+            The Companies House API uses <a
+                href="https://www.example.com/">HTTP Basic
+                Authentication</a> to transmit an API key between the client
+            application and the server. Basic authentication is usually
+            made up of a username and password pair; the Companies House
+            API takes the username as the API key and ignores the
+            password, so can be left blank.
+        </p>
+
+        <h2 class="govuk-heading-l">Example</h2>
+        <h3 class="govuk-heading-m">HTTP Basic Authentication</h3>
+        <p class="govuk-body">
+            For an API key of my_api_key, the following curl request
+            demonstrates the setting of the
+            <code>Authorization</code>
+            HTTP request header, as defined by <a
+                href="https://www.example.com/">RFC2617</a>.
+        </p>
+        <!-- https://tools.ietf.org/html/rfc2617 sticking this link here for easy retrival if we need if when updating the content of this page. -->
+        <section>
+        <pre class="code-block">
+            <code class="">curl -XGET -u my_api_key: https:<span
+                    class="comment">//api.companieshouse.gov.uk/company/00000006</span>
+            </code>
+        </pre>
+        </section>
+        <pre class="code-block">
+            <code class="http">
+                <span class="request">GET <span class="string">/company/00000006</span> HTTP/1.1</span>
```

Comment:
This looks like a close span without an open one. Please run your output page through validator.w3.org and fix errors.

Reply:
They look like they match up to me?

Reply:
I think they match